lib/Expr/Parser.cpp

```diff
@@ -491,10 +494,6 @@ DeclResult ParserImpl::ParseArrayDecl() {
       Error("constant arrays must be completely specified.");
       Values.clear();
     }
-
-    for (unsigned i = 0; i != Size.get(); ++i) {
```

Comment:
Why did you remove this code?  This still needs to be done, right?  So let's keep it at least as a TODO comment.